README.md

```diff
@@ -4,6 +4,10 @@
 
 > Description the project.
 
+- This project is built with html and css.
+- Linters is properly set passing all the test.
+- Header is properly set with links
+- The design is for a mobile view.
```

Comment:
- [ ] The description section of the ReadMe should host the `description` of the project. That is, what the project is about briefly. Most of the ballpoints mentioned here should go to your `pull request summary` and the first ballpoint should go to the `Built With` section in the Readme.